litex/boards/targets/netv2.py

```diff
@@ -72,6 +79,17 @@ def __init__(self, sys_clk_freq=int(100e6), with_ethernet=False, **kwargs):
                 l2_cache_reverse        = True
             )
 
+        with_spi=True
+
+        if with_spi:
+            spi_size = 1024*1024*8
+            spipads = platform.request("spiflash4x")
+            #spipads = platform.request("flash")
+            self.submodules.spiphy = LiteSPIPHY(spipads)
+            self.submodules.spictl = LiteSPI(phy=self.spiphy, endianness=self.cpu.endianness)
+            self.add_memory_region("spixip", self.mem_map["spixip"], spi_size, type="io")
```

Comment:
It's now possible to let the SoC automatically determines the mapping of the added periperal when added like this:
````python3 
spi_xip_region = SoCRegion(origin=self.mem_map.get("spixip",  None), size=spi_size, cached=False)
self.bus.add_slave(name="spixip", slave=self.spictl.bus, region=spi_xip_region)
````
Current code is not updated everywhere, but this is the recommended way for new peripherals. Can you change to that? Also, if you don't need static mapping (which should the case since the bios is using the csr.h and Zephyr/Linux are using Device Tree) you could remove the `spixip` definition in the `mem_map`.